lib/rspec/rails/example/view_example_group.rb

```diff
@@ -13,6 +13,14 @@ module ViewExampleGroup
       include RSpec::Rails::ViewAssigns
       include RSpec::Rails::Matchers::RenderTemplate
 
+      # @private
+      module StubResolverCache
+        def self.resolver_for(hash)
+          @resolvers ||= {}
+          @resolvers[hash] ||= ActionView::FixtureResolver.new(hash)
```

Comment:
My only concern is that this introduces shared state across tests (I know this is by design) but if we were to freeze this object it wouldn't be mutable which would alleviate this concern

Reply:
Sure, that's a straightforward change to make, have pushed with the extra `.freeze`. 👍 

However, there's some further mutable internal state in the resolver that I don't think we'll be able to freeze; it only builds and caches `Template` instances when it receives queries for a specific path, so this happens on first render, not when we're instantiating the resolver.

My intuition is that this shared state should be safe, though. Since we're indexing the cache on the full stub definition (i.e. filename and stubbed contents), and the output of a compiled template should be a function of its contents, I don't think there's any way for the cache to return incorrect values.